node_modules/oae-admin/tenants/js/tenants.js

```diff
@@ -242,11 +242,17 @@ define(['jquery', 'oae.core', 'jquery.jeditable'], function($, oae) {
             var displayName = $.trim($('#tenants-create-displayName', $rootel).val());
             var host = $.trim($('#tenants-create-host', $rootel).val());
 
-            oae.api.admin.createTenant(alias, displayName, host, function(err, tenant) {
+            var opts = {};
+            var emailDomain = $.trim($('#tenants-create-email-domain', $rootel).val());
+            if (emailDomain) {
+                opts.emailDomain = emailDomain;
```

Comment:
Won't *most* tenancies have an email domain? This way it can be added at the same time, without having to add, then search for the tenant and edit.